Bottom: 3b09e16cd2e62e261357a1577fa9c6fc9cfe8dd9
Top:    feded739c897dde9d1bc6462fd1e97616ca34cd1
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-01-04 18:05:04 +0800

Refresh of tun-rx-batching

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 5231602..ac48dd4 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1173,7 +1173,7 @@ static void tun_rx_batched(struct tun_struct *tun, struct tun_file *tfile,
 		struct sk_buff *nskb;
 		local_bh_disable();
 		while ((nskb = __skb_dequeue(&process_queue)))
-			netif_receive_skb(skb);
+			netif_receive_skb(nskb);
 		netif_receive_skb(skb);
 		local_bh_enable();
 	}
